integration-tests/tests/package.json

```diff
@@ -84,6 +84,8 @@
     "@types/chai-as-promised": "^7.1.5",
     "@types/jsrsasign": "^10.5.4",
     "@types/mocha": "^10.0.0",
+    "@types/sinon": "^9.0.5",
+    "assert": "^2.1.0",
```

Comment:
This polyfill seems necessary for electron... will double check

Reply:
I'd like more context on this change.

Reply:
It seemed necessary for sinon but I had since then changed the version to something which has more compatibility so it may not be needed anymore